docs/src/pages/components/radiogroupfield/demo.tsx

```diff
@@ -1,40 +1,65 @@
 import * as React from 'react';
-import { Radio, RadioGroupField, View } from '@aws-amplify/ui-react';
+import {
+  Radio,
+  RadioGroupField,
+  RadioGroupFieldProps,
+} from '@aws-amplify/ui-react';
 
-import { RadioGroupFieldPropControls } from '@/components/RadioGroupFieldPropControls';
-import { useRadioGroupFieldProps } from '@/components/useRadioGroupFieldProps';
+import { Demo } from '@/components/Demo';
+import { RadioGroupFieldPropControls } from './RadioGroupFieldPropControls';
+import { useRadioGroupFieldProps } from './useRadioGroupFieldProps';
 
-export const Demo = () => {
+const propsToCode = ({
+  label,
+  labelPosition,
+  size,
+  direction,
+  name,
+  isDisabled,
+}: RadioGroupFieldProps) => {
+  return (
+    `<RadioGroupField` +
```

Comment:
I'm curious if we only use literal template for the entire returned string?

Reply:
I'm open to suggestions to make these functions to generate code from the demo easier. The one difficulty we run into is if a certain prop is undefined, we don't want to show an empty line in the generated code or else we might get:

```jsx
<RadioGroupField
  label="label"


  direction="row"

>
```

Reply:
gotcha, that make sense